packages/api-client/src/store/workspace.ts

```diff
@@ -178,7 +178,11 @@ const createParamInstance = (param: OpenAPIV3_1.ParameterObject) =>
   createRequestExampleParameter({
     key: param.name,
     value:
-      param.schema && 'default' in param.schema ? param.schema.default : '',
+      param.schema && 'default' in param.schema
+        ? param.schema.default
+        : param.schema && 'examples' in param.schema
+          ? param.schema.examples[0]
```

Comment:
In theory, examples could be an empty array. 🤡

Reply:
it'll just display no value right now, and `__PLANETID__` in the reference example. wdyt?

Reply:
we can probably down the road have a helper function to get default values from openapi spec, i feel like theres a ton of theoretical limits we hit ;)